calculator.go

```diff
@@ -1,13 +1,111 @@
-// Package calculator provides a library for simple calculations in Go.
 package calculator
 
-// Add takes two numbers and returns the result of adding them together.
-func Add(a, b float64) float64 {
-return a + b
+import (
+	"errors"
+	"math"
+	"strconv"
+	"strings"
+)
+
+// Add takes two or more float64 values and returns their sum.
+func Add(a float64, b float64, nums ...float64) float64 {
+	sum := a + b
+	for _, n := range nums {
+		sum += n
+	}
+	return sum
+}
+
+// Subtract takes a variable number of float64 and returns their difference.
+func Subtract(a float64, b float64, nums ...float64) float64 {
+	difference := a - b
+	for _, n := range nums {
+		difference -= n
+	}
+	return difference
+}
+
+// Multiply takes a variable number of float64 and returns their product.
+func Multiply(a float64, b float64, nums ...float64) float64 {
+	product := a * b
+	for _, n := range nums {
+		product *= n
+	}
+	return product
+}
+
+// Divide takes a variable number of float64 and returns their quotient, or
+// an error if the divisor is zero.
+func Divide(a float64, b float64, nums ...float64) (float64, error) {
+	if b == 0 {
+		return 0, errors.New("Cannot divide by zero")
```

Comment:
It's conventional to begin error messages with a lowercase letter: https://github.com/golang/go/wiki/CodeReviewComments#error-strings